games/pacman.puml

```diff
@@ -0,0 +1,153 @@
+@startuml
+package "Process Input Handler" {
+!includesub ../uml/frontend-software-system/process-input-handler.puml!INTERFACE
+}
+package "Render Handler" {
+!includesub ../uml/frontend-software-system/render-handler.puml!INTERFACE
+}
+!include ../uml/frontend-software-system/game-settings.puml
+
+
+package Pac-Man {
+
+class Position {
+ - x : int
+ - y : int
+}
+
+class Wall {
+ - sprite : Sprite
+ - position: Position
+}
+
+class PowerSphere
+{
+ - sprite : Sprite
+ - time : int
+ - position: Position
+ + collect() : int
+ + activatePower() : void
+}
+
+class Sphere
+{
+ - sprite : Sprite
+ - position: Position
+ + collect() : int
+}
+
+class Ghost
+{
+ - sprite : Sprite
+ -state : string
+ -color : string
+ + fleePacman () : void
+ + chasePacman () : void
+}
+
+interface ClydeGhost{}
+interface PinkyGhost{}
+interface InkyGhost{}
+interface BlinkyGhost{}
+
+interface ICharacter
+{
+ - position: Position
+ + respawn (Position position) : void
+ + move (Position) : bool
+}
+class Pacman 
+{
+ - sprite : Sprite
+ - lives : int
+ - direction : string
+ + changeDirection(string direction) : void
+ + looseLife() : bool
+}
+
+class Maze
+{
+ - walls : List<Wall>
+ - spheres: List<Sphere>
+ - powerSphere : List<PowerSpheres>
+ - numberSpheres : int 
+ - numberPowerSpheres : int 
+ + generateMaze() : void
+}
+
+class GameController
+{
+ -maze : Maze
+ - pacman : Pacman
+ -ghosts : List<Ghost>
+ -score : int
+ + start() : void
+ + restart () :void
+ + pause () : void
+ + end () : void
+}
```

Comment:
The PRD doesn't mention anything about game control operations such as: pasuing the game or restarting it.